auth/google_auth.py

```
@@ -447,6 +448,22 @@ def get_credentials(
     """
+    # Check for OAuth access token in environment variables first
+    access_token = os.getenv('GOOGLE_OAUTH_ACCESS_TOKEN')
+    if access_token:
+        logger.info("[get_credentials] Using OAuth access token from environment variable")
+        try:
+            # Create credentials directly from access token
+            credentials = Credentials(token=access_token)
+            
+            # Note: We can't validate scopes for raw access tokens without making an API call
+            # The calling service will handle any scope-related errors
+            logger.debug(f"[get_credentials] Created credentials from environment access token")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)